new/LIT6194MGSatan.xml

```diff
@@ -0,0 +1,108 @@
+                    possessed man from Satan</title>
+                <title xml:id="t2" xml:lang="en">How a demoniac was healed during the Mass which
+                    followed the consecration of the church</title>
+                <title xml:id="t3" xml:lang="la">Daemoniacus liberatur intra liturgiam, quae
+                    consecrationem templi sequitur</title>
+
+                <editor role="generalEditor" key="AB"/>
+                <editor key="SH"/>
+                <funder>Akademie der Wissenschaften in Hamburg</funder>
+            </titleStmt>
+            <publicationStmt>
+                <authority>Hiob-Ludolf-Zentrum für Äthiopistik</authority>
+                <publisher>Die Schriftkultur des christlichen Äthiopiens und Eritreas: Eine
+                    multimediale Forschungsumgebung / Beta maṣāḥǝft</publisher>
+                <pubPlace>Hamburg</pubPlace>
+                <availability>
+                    <licence target="http://creativecommons.org/licenses/by-sa/4.0/"> This file is
+                        licensed under the Creative Commons Attribution-ShareAlike 4.0. </licence>
+                </availability>
+            </publicationStmt>
+            <sourceDesc>
+                <p/>
+            </sourceDesc>
+        </fileDesc>
+        <encodingDesc>
+            <p>A digital born TEI file</p>
+        </encodingDesc>
+        <profileDesc>
+            <creation/>
+            <abstract>
+                <p/>
+            </abstract>
+            <textClass>
+                <keywords>
+                    <term key="ChristianLiterature"/>
+                    <term key="Hagiography"/>
+                    <term key="Miracle"/>
+                </keywords>
+            </textClass>
+            <langUsage>
+                <language ident="en">English</language>
+                <language ident="gez">Gǝʿǝz</language>
+            </langUsage>
+        </profileDesc>
+        <revisionDesc>
+            <change who="SH" when="2020-10-13">Created entity</change>
+        </revisionDesc>
+    </teiHeader>
+    <text>
+        <body>
+            <div type="bibliography">
+
+                <listBibl type="editions">
+                    <bibl>
+                        <ptr target="bm:Budge1930George"/>
+                        <citedRange unit="page">123–126</citedRange>
+                    </bibl>
+                    <bibl>
+                        <ptr target="bm:1968GadTamMalkGiyorgis"/>
+                        <citedRange unit="page">151–153 (third miracle)</citedRange>
+                    </bibl>
+                </listBibl>
+
+                <listBibl type="secondary">
+                    <bibl>
+                        <ptr target="bm:Strelcyn1974CatJRylands"/>
+                        <citedRange unit="page">32</citedRange>
+                        <citedRange unit="item">Ms 12, II.3 (2nd miracle)</citedRange>
```

Comment:
Is there a reason why we have here "2nd" (number) while above "third"?

Reply:
No, I changed later to the numerals and forgot it here